### PR TITLE
Create default HAML style guide

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: config/style_guides/haml.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+linters:
+  AltText:
+    enabled: false
+
+  ClassAttributeWithStaticValue:
+    enabled: true
+
+  ClassesBeforeIds:
+    enabled: true
```
Comment (salbertson) [inline config/style_guides/haml.yml]: Is this a sensible default? https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#classesbeforeids

File: config/style_guides/haml.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  ConsecutiveSilentScripts:
+    enabled: true
+    max_consecutive: 2
+
+  EmptyScript:
+    enabled: true
+
+  HtmlAttributes:
+    enabled: true
```
Comment (salbertson) [inline config/style_guides/haml.yml]: Is this a sensible default? https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#htmlattributes

File: config/style_guides/haml.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  LeadingCommentSpace:
+    enabled: true
+
+  LineLength:
+    enabled: true
+    max: 80
+
+  MultilinePipe:
+    enabled: true
```
Comment (salbertson) [inline config/style_guides/haml.yml]: Is this a sensible default? https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#multilinepipe

File: config/style_guides/haml.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+  RubyComments:
+    enabled: true
+
+  SpaceBeforeScript:
+    enabled: true
+
+  SpaceInsideHashAttributes:
+    enabled: true
+    style: space
```
Comment (salbertson) [inline config/style_guides/haml.yml]: Is this a sensible default? https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#spaceinsidehashattributes

File: config/style_guides/haml.yml (hunk 2 above)
Comment (donokuda) [inline config/style_guides/haml.yml]: Makes sense to me.

File: config/style_guides/haml.yml (hunk 4 above)
Comment (donokuda) [inline config/style_guides/haml.yml]: Makes sense if it's inline with our ruby style guide.

**Comment by donokuda** [comment]
These all make sense to me. @thoughtbot/design any last thoughts?

File: config/style_guides/haml.yml (hunk 1 above)
Comment (gylaz) [inline config/style_guides/haml.yml]: @thoughtbot/design ?

File: config/style_guides/haml.yml (hunk 4 above)
Comment (gylaz) [inline config/style_guides/haml.yml]: Love it.

**Comment by gylaz** [comment]
@salbertson Should the default Hound config be the same as the default linter config? Then we only tweak the one for thoughtbot?

**Comment by salbertson** [comment]
@gylaz, it is the same except for the `RuboCop` setting.

File: config/style_guides/haml.yml (hunk 1 above)
Comment (tysongach) [inline config/style_guides/haml.yml]: I tend to do that…

:+1:

File: config/style_guides/haml.yml (hunk 2 above)
Comment (tysongach) [inline config/style_guides/haml.yml]: :+1:

File: config/style_guides/haml.yml (hunk 4 above)
Comment (tysongach) [inline config/style_guides/haml.yml]: :+1:

**Pull request merged**